src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -73,8 +73,9 @@ void NotationViewInputController::init()
     }
 
     globalContext()->currentMasterNotationChanged().onNotify(this, [this]() {
-        m_isZoomInited = false;
-        initZoom();
+        if (m_isZoomInited == false) {
```

Comment:
Maybe better `if (!m_isZoomInited) {`?

Reply:
On it

Reply:
Seems better in sync with
```
    if (viewport().isValid() && !m_inputController->isZoomInited()) {
        m_inputController->initZoom();
```
in notationpaintview.cpp, which is the only (other) place where this is checked (and only then initialized)